lib/rouge/lexers/make.rb

```diff
@@ -28,15 +37,25 @@ def initialize(opts={})
 
         rule %r/#.*?\n/, Comment
 
-        rule %r/(export)(\s+)(?=[a-zA-Z0-9_\${}\t -]+\n)/ do
+        rule %r/([-s]?include)((?:[\t ]+[^\t\n #]+)+)/ do
```

Comment:
So `include` can be prepended by either `-` or `s`?

Reply:
Yes, either prefix indicates an include operation that fails silently if the file is not found. The two names were introduced in different competing make tools, and now both are often supported.